public/api/conf/2.0/schemas/wco/fileupload/FileUploadResponse.xsd

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns="hmrc:fileupload" xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           targetNamespace="hmrc:fileupload" elementFormDefault="qualified"
+           attributeFormDefault="unqualified">
+
```

Comment:
Should `fileUploadResponse` element name be `FileUploadResponse` to be consistent with `FileUploadRequest`? - A tricky one as Prashant has asked to keep the upscan field names the same (eg `X-Amz-algorithm`) -  what do you think?